### PR TITLE
add TrackHeedSimTool

### PR DESCRIPTION
**Pull request by wenxingfang**
add TrackHeedSimTool to do ionization for drift chamber for issue #177

### CODE REVIEW

File: Simulation/DetSimDedx/CMakeLists.txt
Comment (mirguest) [inline Simulation/DetSimDedx/CMakeLists.txt:4]: It should be avoided to hard code the paths. Please rewrite this part.

**Review by mirguest — CHANGES_REQUESTED** [review]

**Comment by mirguest** [comment]
Hi @wenxingfang ,
I have just created a PR https://github.com/spack/spack/pull/25800 in spack. After the package is merged, I will notify the maintainers of key4hep-spack.
Tao

**Comment by wenxingfang** [comment]
@mirguest I have merged master branch of CEPCSW, but the build is still cancelled.

**Comment by mirguest** [comment]
> @mirguest I have merged master branch of CEPCSW, but the build is still cancelled.

Maybe the spack job failed, then the jobs are cancelled.

**Pull request closed without merging**
